.github/workflows/test.yml

```diff
@@ -0,0 +1,39 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI (test)
+
+# Controls when the workflow will run
+on:
+  workflow_run:
+    workflows: ["name: CI (build)"]
```

Comment:
I actually think we don't need this to be dependent on build. It will fail anyways and it will be easier to manage without the dependency.